src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
Can you please read the issue I filed? The current case is I don't need it somewhere else in the function. If you move it up, it implies it is used somewhere else and it is hard to tell where it is used.

https://github.com/aframevr/aframe/issues/1679